recipe/build.sh

```diff
@@ -0,0 +1,6 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
```

Comment:
I always worry a bit about adding system includes/libs when we don't know what may or may not be available on the system, and I've seen a few cases where the system includes bring something incompatible and break the build.  But I'll leave this to your judgement.

Reply:
@scottwittenburg since this is pure python we can happily yank these lines. They're leftover from when vcs repo used to have C code in it.